shared/src/commonMain/kotlin/com/mirego/kmp/boilerplate/datasource/generic/GenericDataSource.kt

```diff
@@ -0,0 +1,26 @@
+package com.mirego.kmp.boilerplate.datasource.generic
+
+import com.mirego.kmp.boilerplate.datasource.DataSourceUtils
+import com.mirego.trikot.datasources.flow.BaseExpiringExecutableFlowDataSource
+import com.mirego.trikot.datasources.flow.ExpiringFlowDataSourceRequest
+import com.mirego.trikot.datasources.flow.FlowDataSourceRequest
+import kotlinx.serialization.KSerializer
+import kotlinx.serialization.json.Json
+import kotlin.time.Duration.Companion.minutes
+
+open class GenericDataSource<T>(
```

Comment:
Shouldn't these generic datasources be provided directly in the `com.mirego.trikot.datasources` library?

Reply:
Agreed that we need those in the lib